tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -430,6 +450,36 @@ def docomplicated(suptitle=None):
 example_plot(axs[0], fontsize=12)
 axs[1].set_position([0.2, 0.2, 0.4, 0.4])
 
+###############################################################################
+# .. _compressed_layout:
+#
+# Grids of fixed aspect-ratio Axes: "compressed" layout
+# =====================================================
+#
+#
+# By default, ``constrained_layout`` acts on the original position of an axes.
+# However, when Axes have fixed aspect ratios, one side is made shorter.  For
+# a grid of such Axes, this can leave a large gap in the shortened direction.
+# In the following the Axes are square, but the figure quite wide so there is
+# a horizontal gap:
+
+fig, axs = plt.subplots(2, 2, figsize=(5, 3),
+                        sharex=True, sharey=True, layout='constrained')
+for ax in axs.flat:
+    ax.imshow(arr)
+fig.suptitle('Constrained layout: fixed-aspect plots')
```

Comment:
Using the keyword notation may make the connection the code even stronger:
```suggestion
fig.suptitle("fixed-aspect plots, layout='constrained'")
```